src/swap/types.ts

```diff
@@ -45,6 +45,7 @@ export type GetQuoteAPIParams = {
   to: AddressOrETH | ''; // The destination address or 'ETH' for Ethereum
   amount: string; // The amount to be swapped
   amountReference?: string; // The reference amount for the swap
+  v2Enabled?: boolean; // Whether to use V2 of the API (default: false)
```

Comment:
why not just enable it as default? After all it should not have a real change for the experience, right?

Reply:
see Slack

Reply:
use `experimental`

Reply:
`v2Enabled` this is such a bad way to name a property for an API.

Reply:
do we really need this year?